rpcs3/Emu/RSX/RSXZCULL.cpp

```diff
@@ -323,9 +325,13 @@ namespace rsx
 				break;
 			}
 
-			rsx::reservation_lock<true> lock(sink, 16);
-			auto report = vm::get_super_ptr<atomic_t<CellGcmReportData>>(sink);
-			report->store({ timestamp, value, 0 });
+			{
+				rsx::reservation_lock<true> lock(sink, 16);
+				auto report = vm::get_super_ptr<atomic_t<CellGcmReportData>>(sink);
+				report->release({ timestamp, value, 0 });
+			}
+
+			vm::reservation_notifier(sink).notify_all(-128);

```

Comment:
I'm pretty sure this is undesired. notify_all is slow and we often have to pause everything to drain a large number of zcull objects.